README.md

```diff
@@ -37,6 +37,8 @@ Enable automatic status/favourite deletion with config options. Example:
 delete_older_statuses = true
 # Delete Mastodon favourites that are older than 90 days
 delete_older_favs = true
+# Do not sync boosts (reblogs)
+ignore_boosts = true
```

Comment:
let's set this to false by default, which is the current behavior.

Hm, now that I think about it that would be a double negation of "ignore" and then "false" which is not ideal.

I think we should rather call it `sync_reblogs = true` and `sync_retweets = true`? Also agreeing on reblog if that is the official Mastodon wording.

Reply:
> let's set this to false by default, which is the current behavior.

Will do.

> Hm, now that I think about it that would be a double negation of "ignore" and then "false" which is not ideal.

Good point, I am not a fan of double negations either.
 
> I think we should rather call it `sync_reblogs = true` and `sync_retweets = true`? Also agreeing on reblog if that is the official Mastodon wording.

Perfect! 👌